src/utils.jl

```diff
@@ -1,3 +1,20 @@
+"""
+    pick_chunksize(totalsize, mode, ftype, argtypes...)
+
+Return a reasonable chunk size for batched differentiation.
+
+!!! warning
+    This function is experimental, and not part of the public API.
+"""
+function pick_chunksize(
+    totalsize::Integer,
+    mode::Mode,
+    ftype::Type,
+    argtypes::Vararg{Type{<:Annotation}, Nargs}
```

Comment:
I think also you'll need the return type annotation here too

Reply:
done in the last commit